.github/workflows/ci.yml

```diff
@@ -0,0 +1,41 @@
+name: CI
+on: [push]
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Setup node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 14.17.x
+      - name: Yarn install
+        uses: borales/actions-yarn@v4
+        with:
+          cmd: install
+      - run: yarn check
+      - run: yarn lint
+      - run: yarn flow
+      - run: yarn compile
+      - run: yarn test
```

Comment:
Optional, but might be a nicer dev experience if you had a test reporter after a step like this, to send results to github, like [here](https://github.com/guardian/janus/actions/runs/5726094858/job/15515885589)
Just means you can see the output of the tests a little nicer than having to scroll through the logs, and github might make improvements to that UI in the future.
You can see the instructions here for one that I've used with Janus and other repos
https://github.com/dorny/test-reporter
**Note**: under permissions at the top, you'll need to add
```yaml
checks: write
```